grails-app/utils/org/pih/warehouse/CsvUtil.groovy

```diff
@@ -0,0 +1,28 @@
+package org.pih.warehouse
+
+class CsvUtil {
+    static final Set<String> validBooleanValues = ['t', 'f', 'true', 'false', '1', '0', 'y', 'n', 'yes', 'no']
+    static final Set<String> trueBooleanValues = ['t', 'true', '1', 'y', 'yes']
```

Comment:
nitpick: please use all caps for constants: `VALID_BOOLEAN_VALUES`, `TRUE_BOOLEAN_VALUES`